brain/neuron/morphology.cpp

```diff
@@ -125,7 +125,7 @@ Sections Morphology::getSections( const SectionTypes& types ) const
 
 Section Morphology::getSection( const uint32_t& id ) const
 {
-    if(( *_impl->types )[id] == SECTION_SOMA )
+    if(( *_impl->types )[id] == brion::enums::SECTION_SOMA )
```

Comment:
Because then I have to cast ( *_impl->types )[id] to the new enum. I don't want to use the new type in the member variable because that requires an interpret cast of shared_ptr<vector<SectionTypes> > from brion enum to brain enum or copying the whole array.
It really annoys me how it looks now, but that's the best that I could find.